### PR TITLE
Resolved #2109 This new script should update asset panels, based on c…

### LINKED ISSUE #2109: New script: ACTIONS - ASSET PANEL UPDATER
This new script should update asset panels, based on changes reported or new information at intake. This is part of the ongoing Project Bowie.

Any takers? Could be a simple way to add some ACTIONS scripts to your repertoire!

### PR DESCRIPTION
**Pull request by StormageddonDLOA**
…hanges reported or new information at intake.

This script will either add a new ACCT panel to MAXIS or update an existing ACCT panel. 
This is part of the ongoing Project Bowie.

### CODE REVIEW

File: Script Files/ACTIONS/ACCT PANEL UPDATER.vbs [hunk 1]
```diff
@@ -0,0 +1,494 @@
+'Required for statistical purposes==========================================================================================
+name_of_script = "ACTIONS - ACCT PANEL UPDATER.vbs"
```
Comment (theVKC) [inline Script Files/ACTIONS/ACCT PANEL UPDATER.vbs]: Your file name doesn't match the `name_of_script`. These need to match, and the name needs to meet the criteria specified in the project readme.

File: Script Files/ACTIONS/ACCT PANEL UPDATER.vbs [hunk 2]
```diff
@@ -0,0 +1,494 @@
...
+		'checks if ratio one has been entered.
+		IF ratio_one = "" THEN err_msg = err_msg & vbCr & "You must enter the ACCT ratio."
+		'checks if ratio two has been entered.
+		IF ratio_two = "" THEN err_msg = err_msg & vbCr & "You must enter the ACCT ratio."
+		'popups the list of errors that need to be fixed.
+		IF err_msg <> "" THEN MsgBox "*** NOTICE!!! ***" & vbCr & err_msg & vbCr & vbCr & "Please resolve for the script to continue."
+	LOOP UNTIL err_msg = ""
+
+	EMWriteScreen "nn", 20, 79
+	'checking to see if a new panel has been created
```
Comment (theVKC) [inline Script Files/ACTIONS/ACCT PANEL UPDATER.vbs]: This looks like it's just deciding it needs a new panel. I don't see a corresponding `EMReadScreen`. How is it checking to see if the panel was created?

File: Script Files/ACTIONS/ACCT PANEL UPDATER.vbs [hunk 3]
```diff
@@ -0,0 +1,494 @@
...
+
+else
+
+call check_for_MAXIS(true)
+
+	'navigates to STAT/ACCT
+	EMWriteScreen "stat", 16, 43
+	EMWriteScreen "acct", 21, 70							
+	EMWriteScreen MEMB_num, 21, 75
+msgbox "hi"
```
Comment (theVKC) [inline Script Files/ACTIONS/ACCT PANEL UPDATER.vbs]: This looks like a temp MsgBox.

**Comment by theVKC** [comment]
This looks pretty good @StormageddonDLOA! Save the few comments I put on your file, I think it's ready for use.

Once you fix the above issues, I think it might be a good idea for you to deploy this at Stearns in Agency Custom. Considering the June release is due now, I'd rather this get some more time for testing. Perhaps after a couple of weeks of Stearns testing, you may feel it's ready to be released as-is or you could get some good feedback on ways to improve it (I can't think of anything right now, but you might as folks use it).

**Comment by theVKC** [comment]
@StormageddonDLOA, I'm going to close this pull request. Your file name still isn't correct (needs "ACTIONS - " in front of the actual file name.

In any case, I want Stearns to test this in Agency Custom for a couple of weeks before you pull it. You can drop the script file into your agency custom folder for testing it out. I think this will be a good way to make sure it's acting like you want: just tell your users to go there to run the script, and ask them to provide you with feedback.

**Comment by StormageddonDLOA** [comment]
It DID say ACTIONS at first so I thought that must be what’s wrong.


In Script Files/ACTIONS/ACCT PANEL UPDATER.vbs<https://github.com/MN-Script-Team/DHS-MAXIS-Scripts/pull/2363#discussion_r67691506>:

> @@ -0,0 +1,494 @@

> +'Required for statistical purposes==========================================================================================

> +name_of_script = "ACTIONS - ACCT PANEL UPDATER.vbs"

Your file name doesn't match the name_of_script. These need to match, and the name needs to meet the criteria specified in the project readme.
From the email you sent me originally.

Tim

**Pull request closed without merging**